src/document/docUtil.ts

```diff
@@ -39,6 +39,10 @@ export function getSelection(): Selection | null {
     // We only care about the last range
     let range = selection.getRangeAt(selection.rangeCount - 1);
 
+    // Changes to the text layer in PDFjs v2.14.305 require the start range to be set manually for correct highlighting behavior
```

Comment:
Is this backward compatible with the older versions of PDF.js? We still use v2.2.228 for IE11.

Reply:
Yup! I just checked and it works on IE11 as well!